[DOCS-7579] NDM profile onboarding

aliciascott opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [ ] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+| No Modification | The device's returned value will be used directly as the tag value. |
+| Format | This can be [mac_address][5] or [ip_address][6]. |
+| Extract Value | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |
+| Mapping | This is described [here][8]. Note: our current docs may imply this can ONLY be used for table tags, which needs correction. |
```
How long will it take to correct this? Since users won't know this is separate docs, sounds like we're discrediting ourselves. Recommend removing and just fixing the incorrect statement.

```suggestion
| Mapping | This is described [here][8]. |
```

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+
+1. Select whether the tag value originates from an `OID` value or a segment of the table index. If `Index` is chosen as the source, an index position must be specified, which then becomes the tag.
+
+<details>
```
I really like how this looks! I wonder if this can also indent so it's in line with number 1.

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+
+### Apply a profile to created devices
+
+1. Once you've saved a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you've created. 
```
```suggestion
1. After you save a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you created. 
```

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+### Apply a profile to created devices
+
+1. Once you've saved a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you've created. 
+2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agent's.
```
```suggestion
2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agents.
```

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+1. Once you've saved a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you've created. 
+2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agent's.
+3. Restart the Datadog Agent.
+4. To ensure the profiles you've created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.
```
```suggestion
4. To ensure the profiles you created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.
```

estherk15 commented on content/en/network_monitoring/devices/guide/device_profiles.md:
```diff
@@ -0,0 +1,145 @@
+3. Restart the Datadog Agent.
+4. To ensure the profiles you've created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.
+
+**Need updated screen shot** - This is a placeholder
```
Flagging placeholder

estherk15 reviewed (commented):
There are two placeholders for images, recommend removing the placeholder and images and update later. Also, one of the anchor links wasn't redirecting. Let me know if you want a rereview!

aliciascott commented:
Hi @estherk15 I updated the screenshots and the rest of the items for your review, also fixed the broken anchor and indented the dropdown list, can you do a re-review ? Thanks!

estherk15 reviewed (approved):
Looks good, thanks for taking my suggestions!
